### PR TITLE
fixing create mapping and style ref #35

### PR DESCRIPTION
**Pull request by fabianazioti**

### CODE REVIEW

**Review by raphaelrpl — APPROVED** [review]

File: examples/available_mappings.py [hunk 1]
```diff
@@ -10,9 +10,9 @@
 from lccs import LCCS
 
 # Change to the LCCS-WS URL you want to use.
-service = LCCS("http://brazildatacube.dpi.inpe.br/dev/lccs/")
+service = LCCS("'https://brazildatacube.dpi.inpe.br/lccs/")
```
Comment (M3nin0) [inline examples/available_mappings.py:13]: You may need to remove the quote

File: examples/classification_system_metadata.py [hunk 2]
```diff
@@ -10,11 +10,11 @@
 from lccs import LCCS
 
 # Change to the LCCS-WS URL you want to use.
-service = LCCS("http://brazildatacube.dpi.inpe.br/dev/lccs/")
+service = LCCS("http://brazildatacube.dpi.inpe.br/lccs/")
```
Comment (M3nin0) [inline examples/classification_system_metadata.py:13]: HTTPS ?

File: examples/classification_systems.py [hunk 3]
```diff
@@ -10,7 +10,7 @@
 from lccs import LCCS
 
 # Change to the LCCS-WS URL you want to use.
-service = LCCS("http://brazildatacube.dpi.inpe.br/dev/lccs/")
+service = LCCS("http://brazildatacube.dpi.inpe.br/lccs/")
```
Comment (M3nin0) [inline examples/classification_systems.py:13]: HTTPS

File: examples/ex-01.py [hunk 4]
```diff
@@ -12,11 +12,11 @@
 
 print(lccs.__version__)
 
-url = os.environ.get('LCCS_SERVER_URL', 'https://brazildatacube.dpi.inpe.br/dev/lccs/')
+url = os.environ.get('LCCS_SERVER_URL', 'https://brazildatacube.dpi.inpe.br/lccs/')
 
-service = lccs.LCCS('http://127.0.0.1:5000/')
+service = lccs.LCCS(url)
```
Comment (M3nin0) [inline examples/ex-01.py:17]: maybe insert a string with server url text (e. g. "server url")

File: examples/style.py [hunk 5]
```diff
@@ -13,7 +13,7 @@
 service = LCCS("http://brazildatacube.dpi.inpe.br/dev/lccs/")
```
Comment (M3nin0) [inline examples/style.py:13]: HTTPS

**Review by M3nin0 — CHANGES_REQUESTED** [review]

File: tests/tests_lccs/jsons/class.json [hunk 6]
```diff
@@ -1,19 +1,16 @@
...
     {
-      "href": "http://localhost:5000/classification_system/PRODES/classes/Floresta",
+      "href": "http://localhost:5000/classification_systems/1/classes/2",
       "rel": "self",
       "title": "Link to this document",
       "type": "application/json"
     },
     {
-      "href": "http://localhost:5000/classification_systemPRODES/classes",
+      "href": "http://localhost:5000/classification_systems1/classes",
```
Comment (mzaglia) [inline tests/tests_lccs/jsons/class.json:13]: classification_systems/1/classes ?

**Review by mzaglia — CHANGES_REQUESTED** [review]

**Review by mzaglia — APPROVED** [review]

**Review by M3nin0 — APPROVED** [review]

**Review by raphaelrpl — APPROVED** [review]

**Pull request merged**